.github/workflows/pr-coauthor.yml

```diff
@@ -0,0 +1,17 @@
+name: Write coauthors to a pull request
```

Comment:
```suggestion
name: Add coauthors to a pull request
```

Reply:
I am not exactly sure about 'add', since the workflow isn't automated enough to add co-authors to the merged PRs. It adds a comment to the pull request for maintainers to add them. I wouldn't want to be misleading with the description.

Reply:
Makes sense 😄